articles/tutorials/how-to-update-applications-client-secret.md

```diff
@@ -16,11 +16,11 @@ Once you have changed your application's `client_secret` you will need to update
 
 ## How to generate a new client_secret
```

Comment:
Sorry, I don't have access to that repo. Do you know if that PR is to be merged soon? I'm wondering whether I should add this to the doc now or later on.